### ISSUE TITLE
Revert move and rename of jenkins-infra/governance-meetings

### ISSUE THREAD

**Issue by daniel-beck** [jenkins-infra/helpdesk#3277]
> I would recommend it be called governance not governance-meeting so it can be used for more things, like maybe sponsorship details.

It's an archive of old stuff, with no reason to make it a living repo. Just create a new repo for what you need. We're not merging `confluence-data` into `jenkins.io` either, after all.

>  Can be closed if this is resolving to jenkinsci.github.io and is still working:
> https://meetings.jenkins-ci.org/

This alone is an indicator the repo should not be used for other stuff, because it's pretty tied to the domain.

_Originally posted by @daniel-beck in https://github.com/jenkins-infra/helpdesk/issues/3271#issuecomment-1332297284_

**Comment by daniel-beck** [comment]
Also https://github.com/jenkinsci/governance/pull/9.

**Comment by timja** [comment]
confluence-data is different in that it's wiki content that anyone could contribute to.

Why do old meetings need to go in a different repository from new meetings?

**Comment by daniel-beck** [comment]
> Why do old meetings need to go in a different repository from new meetings?

They don't, necessarily, but neither move nor rename are related to not holding new meeting logs. You did both in response to https://github.com/jenkins-infra/helpdesk/issues/3271#issuecomment-1331809145

**Comment by dduportal** [comment]
Not sure what is the status there: it does not seem that a consensus was found?

**Comment by daniel-beck** [comment]
Well, again, neither move nor rename are needed (i.e., can be reverted) as long as we agree to focus the repo on meeting logs. If we don't, the domain doesn't make sense, and then all the GitHub pages stuff doesn't make sense either.

Whether it's an archive for existing old logs, or whether we add new logs is secondary.

**Comment by dduportal** [comment]
> I don't think it is healthy for the project when things get implemented without discussion or waiting for feedback (6 hours in a ticket without pings isn't "waiting"). Yet that is what happened here. And now "no consensus" is enough to block the reversal?

+1 with this: the subject in itself is not strategic matter but it raises question in the method.

Given that the "root cause" here is the initial discussion (ref. https://github.com/jenkins-infra/helpdesk/issues/3271#issuecomment-1331765610 and https://github.com/jenkins-infra/helpdesk/issues/3271#issuecomment-1331443500) about where to put the repository for new governance needs, let's raise it again with the outcomes of our exchanges.

The following definition, proposed by @daniel-beck could help to "decide" why a repository would go under `jenkinsci` or `jenkins-infra` (because same kind of question for repositories such as jenkins-infra/jenkins.io for instance):

> - jenkinsci is stuff that ends up as deliverables to end users
> - jenkins-infra is stuff that’s not


=> It's quite simple and not focused on the "responsability" but rather on the "target audience" which is easier to underline.

Under that definition, I'll move (or ask to) move the repository jenkinsci/governance-meetings back to infra, and reopen #3271 to re-evaluate properly (as in "we did it too hastily, let's do it again") the need.

Is there any objection?

Unless someone disagree in the next 24 hours,

**Comment by timja** [comment]
FYI I've moved it back although moving it has removed my permissions so I can't rename it

https://github.com/jenkins-infra/governance

**Comment by timja** [comment]
@lemeurherve has restored permissions and renamed back to https://github.com/jenkins-infra/governance-meetings

**Comment by timja** [comment]
DNS updated, 1 hour TTL

**Comment by dduportal** [comment]
Domain verified, TXT records cleaned up from Azure portal (there was already one but marked as invalid by GitHub: did you created it @timja?)

<img width="995" alt="Capture d’écran 2022-12-07 à 13 24 43" src="https://user-images.githubusercontent.com/1522731/206178894-08be379e-6746-452c-b61e-7018c0f8bbaa.png">

**Comment by dduportal** [comment]
```
To make sure your custom domain remains verified, keep the TXT record in your domain's DNS configuration.
```

Damn I'm so dumb 🤦

**Comment by dduportal** [comment]
Re-verified and kept the TXT. Let's wait 1 hour and see the DNS propagation.

**Comment by timja** [comment]
> there was already one but marked as invalid by GitHub: did you created it
no I don't have access to jenkins-infra ones but as you found out it will have been from someone 'cleaning' it up before

**Comment by dduportal** [comment]
Damn, this GH page stuff is not working at all and keep complaining about domain.

- Disabled GH page at all
- Repo renamed to https://github.com/jenkins-infra/governance-meetings-archives to clean up GH page

I'm now trying to re-deploy it with the domain

**Comment by dduportal** [comment]
GitHub is completly broken. I can't define a custom domain for this repository. Let's wait for 24 hours as per their documentation. Otherwise we'll have to contact their support

**Comment by dduportal** [comment]
Still failing: opened an issue to GitHub support (ID #1910509)

**Comment by dduportal** [comment]
@timja @NotMyFault (or any admin of the `jenkinsci` organization) could you make sure that the domain `meetings.jenkins-ci.org` is **absent** from https://github.com/organizations/jenkins-infra/settings/domains please?

**Comment by dduportal** [comment]
> someone must've deleted it: ![image](https://user-images.githubusercontent.com/21194782/206686113-c8e494cc-3111-43a1-8ca3-475da79758ae.png)

My bad @timja , I meant https://github.com/organizations/jenkinsci/settings/domains (I've edited my comment).

I removed the verification from `jenkins-infra`. But there might be a leftover in `jenkinsci`.

**Comment by timja** [comment]
I noticed that and updated my url that screenshot is accurate

**Comment by dduportal** [comment]
Thanks @timja you rocks ❤️

**Comment by NotMyFault** [comment]
> @timja @NotMyFault (or any admin of the `jenkinsci` organization) could you make sure that the domain `meetings.jenkins-ci.org` is **absent** from ~https://github.com/organizations/jenkins-infra/settings/domains~ https://github.com/organizations/jenkinsci/settings/domains please?

I'm not an admin of the jenkinsci org 👀

**Comment by dduportal** [comment]
- https://meetings.jenkins-ci.org/ is back
- Its source is in https://github.com/jenkins-infra/governance-meetings-archives

TL;DR;: there 2 different "domain validations" in organization settings and I was using the "Security" one, while the required one was the "Pages" one. GH Support helped me.
